### DISCUSSION TITLE
[HELP] Can't access Web Advance Config on N64 Blueretro (Retro Scaler)

### DISCUSSION THREAD

**Discussion by FerrazCezar** [darthcloud/BlueRetro#589] [General]
Hi,

I'm having problems to connect my RetroScaler N64 Blueretro adapter to the Web interface that allows change the configuration (BlueRetro Advance config). I can find my adapter and pair, but after that this message of error appears:

> Reading App version...
> Reading Api version...
> Requesting Bluetooth Device...
> Connecting to GATT Server...
> Getting BlueRetro Service...
> ABI version: 2
> Init Cfg DOM...
> Get Global Config CHRC...
> Reading Global Config...
> Argh! NotSupportedError: GATT operation failed for unknown reason.
> Bluetooth Device disconnected

I've tried with the bluetooth dongle on my desktop, tried with my laptop and even the phone, but nothing works. Also tried on Chrome and Edge, but the same error appears. Anyone know a solution for this?

**Comment by h4x0rJyNx** [comment]
I delete my cfg file and it started working.
go here https://blueretro.io/files.html connect and delete cfg

**Reply by FerrazCezar** [reply]
Thanks for the reply, but I can't even connect to this. The message is the following:

> Reading App version...
> Requesting Bluetooth Device...
> Connecting to GATT Server...
> Device name: BlueRetro_N64_18EA
> Device id: k5ikzHVnIVgzRASSmwJ9mQ==
> Getting BlueRetro Service...
> Argh! NotSupportedError: GATT Error Unknown.
>  Bluetooth Device disconnected

**Comment by FerrazCezar** [comment]
Sorry for bumping my post, but I want to share a solution that I found. I managed to update my stock firmware (v1.8_hw1) to the latest (v1.8.2_hw1) through a web interface that I found here: [https://blueretro.delisle.io](https://blueretro.delisle.io).

I used my android phone and the firmware update went smooth. After that I was able to access the proper configs through the [https://blueretro.io](https://blueretro.io) web interface.

Hope you guys find this info useful, since there's some people having problems to connect their blueretro adapters to the web interface.
